portal-web/test/functional/com/liferay/portalweb/paths/portlet/mediagallery/MediaGallery.path

```diff
@@ -91,6 +91,11 @@
 	<td>//li/a[contains(@class,'carousel-menu-item') and contains(@class,'carousel-menu-index')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SCREENSHOTS_FJORD_THEME_PNG</td>
```

Comment:
`SCREENSHOTS_THEME_HOME_PAGE_PNG` helps describe the image better.

Also, I don't think this quite fits in MediaGallery.path
Let's add a new path file in /portalweb/paths/portlet/theme/FjordTheme.path